### DISCUSSION TITLE
Does anyone know how to do animations with xmls (Like FNF)

### DISCUSSION THREAD

**Discussion by Yemdff** [HaxeFlixel/flixel#2527] [Q&A]
The title explains

**Comment by Geokureli** [comment]
FNF uses [FlxAtlasFrames](https://github.com/HaxeFlixel/flixel/blob/dev/flixel/graphics/frames/FlxAtlasFrames.hx). which it loads [here](https://github.com/ninjamuffin99/Funkin/blob/master/source/Paths.hx#L108-L113) and applies to sprites, [like this](https://github.com/ninjamuffin99/Funkin/blob/master/source/GitarooPause.hx#L28)
